client/blocks/jetpack-benefits/general-benefits.tsx

```diff
@@ -108,7 +94,7 @@ const JetpackGeneralBenefits: React.FC< Props > = ( props ) => {
 	}
 
 	// Unlimited Video Hosting
-	if ( hasBusinessPlan || hasSecurityRealTimePlan || hasCompletePlan ) {
+	if ( planHasFeature( productSlug, WPCOM_FEATURES_VIDEOPRESS ) ) {
```

Comment:
Which feature should this be? It doesn't look like VideoPress is right.

Reply:
I don't think unlimited video is something jetpack offers at all anymore. My inclination would be to keep the feature check and change the text to indicate 1TB of video storage.

References:

- pxWta-Zz-p2
- p7pQDF-6l3-p2
- https://github.com/Automattic/wp-calypso/pull/56103/files#diff-aae1f02893cf95f6f8e13bc3e73bd81d0548c4a58a923c8f4240bc6948255ff1L124